.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+branch = True
```

Comment:
Given that it's only a finger-in-the-air way of enforcing a reasonable confidence in the test suite I don't have any great issue with treating code lines as the only metric we care about to start with.

If we reached 100% line coverage then I might consider branch coverage too.